test/app/submissions_test.rb

```diff
@@ -106,6 +106,23 @@ def test_unlike_a_submission
     post "/submissions/#{submission.key}/unlike", {}, login(bob)
   end
 
+  def test_unlike_removes_unread_likes
+    fred = User.create({username: 'bob', github_id: 3, email: "bob@example.com"})
```

Comment:
I really liked when line comments disappeared on change... Why, GitHub!?? Why???? (Not that I don't like all the other new fancy features).